Project.toml

```
@@ -11,7 +11,7 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 
 [compat]
 Conda = "1.3"
-Memento = "0.12"
+Memento = "1.0"
```

Using `1.0` limits us to `1.0.x`
```suggestion
Memento = "1"
```

---

That is incorrect, but I still prefer your suggestion